hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -276,5 +276,11 @@ module clkmgr import clkmgr_pkg::*; (
   // Assertions
   ////////////////////////////////////////////////////
 
+  `ASSERT_KNOWN(TlDValidKnownO_A,  tl_o.d_valid     )
+  `ASSERT_KNOWN(TlAReadyKnownO_A,  tl_o.a_ready     )
+% for intf in export_clks:
+  `ASSERT_KNOWN(ExportClocksKownO_A, clocks_${intf}_o)
+% endfor
```

Comment:
One QQ is that the clock should be known even with reset assertion. Maybe we should define this without defining the reset? Currently `prim_assert_standard_macros.svh` does not define that kind of macro.

Reply:
oooo... that's a good point....

Reply:
i've created an issue for this one.